easybuild/easyconfigs/c/CP2K/CP2K-4.1-foss-2016b-psmp.eb

```diff
@@ -0,0 +1,43 @@
+name = 'CP2K'
+version = '4.1'
+versionsuffix = '-psmp'
+
+homepage = 'http://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = [SOURCEFORGE_SOURCE]
+
+patches = [
+#    'CP2K-2.6.0-ifort-compiler-bug-fix.patch',
+    'CP2K-2.4.0-fix_compile_date_lastsvn.patch',
+]
+
+dependencies = [
+    ('Libint', '1.1.6'),
+    ('libxc', '2.2.3'),
+    ('PLUMED', '2.2.3'),
+    ('libxsmm', '1.4.4'),
```

Comment:
@ocaisa any reason you're sticking to libxsmm 1.4.4, while there's a 1.6.4 available too (I'll update #3890 to use 1.6.4)